rfc/github-cdevents.md

```diff
@@ -0,0 +1,58 @@
+[github-webhook](https://docs.github.com/en/webhooks/using-webhooks/creating-webhooks) page.
+
+### CDEvent github-webhook-tranlator 
+
+A new REST endpoint `/github-events` will be implemented to receive all github-events and translate them to CDEvents.
+The below list of [CDEvents-Source Code Control Events](https://github.com/cdevents/spec/blob/v0.3.0/source-code-version-control.md) will be mapped with corresponding Github-Webhook event types to translate.  
+Since this effort is to process [webhook events](https://docs.github.com/en/webhooks/webhook-events-and-payloads), we consider processing the webhook-events 
+related to branch, repository, deployment and artifact. The [CDEvent-Spec](https://github.com/cdevents/spec/blob/v0.3.0/spec.md) does not have events for tags or release yet.
+
+
+| CDEvent Type                     | Webhook/Github Event Type   | Comments                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+|:---------------------------------|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| dev.cdevents.repository.forked   | fork                        | A user forks a repository. (should be defined in the spec)                                                                                                                                                                                                                                                                                                                                                                                                        |
+| dev.cdevents.repository.imported | repository_import           | when a new repository is imported to Github. (should be defined in the spec)                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I wonder if we could use `repository.created` here too.
I had never even heard of a repository import functionality before 😅